portality/scripts/out.csv

```diff
@@ -0,0 +1,19 @@
+J,o,u,r,n,a,l, ,8,5,d,4,1,b,c,b,8,7,8,c,4,1,9,d,8,c,b,7,b,3,3,8,7,f,a,2,9,8,7,d, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,b,9,e,3,2,b,4,4,9,b,5,9,4,b,2,8,b,d,4,f,f,3,4,c,5,e,0,7,6,2,7,1, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,3,3,f,2,9,f,8,7,6,2,f,1,4,8,7,7,8,4,2,2,4,3,9,d,1,b,0,c,4,2,f,2, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,0,a,5,4,d,8,d,2,3,6,2,b,4,1,6,3,a,5,e,4,4,d,a,d,3,7,b,0,0,f,0,e, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,3,3,8,6,8,c,1,5,f,d,9,f,4,5,2,f,a,2,e,8,6,d,9,7,f,3,8,5,4,5,8,5, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,4,5,8,6,d,1,0,2,0,e,9,e,4,9,5,3,8,3,e,7,d,0,3,8,1,b,e,9,b,1,d,5, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,1,8,f,3,1,5,9,c,0,7,f,b,4,b,1,7,a,8,f,3,7,2,0,7,e,9,1,0,6,8,9,e, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,6,1,d,9,5,f,6,f,8,8,4,2,4,e,8,c,a,9,5,2,a,d,4,e,f,0,f,f,f,a,3,3, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,6,e,b,5,e,6,7,5,c,3,b,9,4,e,9,e,a,6,6,a,e,8,a,7,1,c,a,d,f,d,4,f, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,5,2,c,8,a,b,6,1,5,a,b,6,4,f,1,7,a,1,1,d,1,9,e,0,c,c,1,6,1,0,e,0, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,5,d,a,3,a,4,1,f,6,b,d,0,4,6,b,e,9,b,f,e,0,f,8,8,d,4,c,d,7,d,4,e, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,e,d,5,d,0,3,2,0,a,7,f,6,4,3,c,0,a,e,9,8,8,3,1,9,c,6,8,7,1,d,d,4, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,e,a,c,c,e,f,d,5,d,4,1,d,4,5,3,5,a,3,5,6,a,5,0,5,3,0,2,1,a,b,c,1, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,9,7,c,6,9,b,a,5,9,d,3,7,4,8,d,f,9,b,8,6,5,2,e,2,2,5,4,0,1,d,c,5, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,c,8,c,7,b,b,7,0,a,0,b,4,4,d,b,d,8,5,7,7,3,a,1,b,8,1,c,d,a,3,e,e, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,6,5,b,7,d,4,4,9,f,a,7,c,4,4,0,4,9,2,6,f,8,2,d,2,a,6,2,0,6,0,a,4, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,1,c,9,e,f,7,b,4,7,a,0,d,4,a,9,f,b,3,b,c,5,3,d,b,b,5,1,4,a,3,9,b, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,e,a,f,4,2,8,5,8,f,4,0,1,4,f,0,1,9,5,f,1,c,1,5,8,4,1,2,b,c,a,e,9, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.
+J,o,u,r,n,a,l, ,3,5,e,7,f,1,3,4,3,c,7,3,4,f,0,9,b,6,4,c,1,6,7,8,a,7,c,d,4,a,e,2, ,m,o,d,i,f,i,e,d, ,a,n,d, ,s,a,v,e,d,.

```

Comment:
Something weird about this CSV format :)

Reply:
Oh boy.....